package.json

```diff
@@ -5,52 +5,29 @@
   "author": "Antony Jones",
   "license": "MIT",
   "scripts": {
-    "dev": "vite dev",
-    "build": "vite build",
+    "dev": "vite dev --host",
+    "build": "vite build --config vite.components.config.js",
     "preview": "vite preview",
-    "package": "svelte-kit sync && svelte-package && publint",
-    "prepublishOnly": "npm run package",
     "check": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json --watch",
     "lint": "eslint .",
     "test": "mocha \"src/**/*.spec.js\""
   },
-  "exports": {
-    ".": {
-      "types": "./dist/index.d.ts",
-      "svelte": "./dist/index.js"
-    },
-    "./Banner.svelte": "./dist/Banner.svelte",
-    "./style.css": "./dist/style.css"
-  },
   "files": [
-    "dist",
-    "!dist/**/*.spec.*"
+    "src/lib/sdk",
+    "dist/v1/sdk"
```

Comment:
this still doesn't make any sense to me, we don't need a versioned dist dir, because the package has a version - it's an npm package

Reply:
Resolved 6e7380ecb1aff92a6fb16a2f9ae2f515d6480a09